pkg/hive/pb/hive.proto

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+syntax = "proto3";
+
+package pb;
+
+message Subscribe {
+    uint32 Depth = 1;
+    bool PeersResponse = 2;
```

Comment:
i initially did not realise what this thing does but i really would like to see this removed. overengineered